### ISSUE TITLE
Media Player(s) from HomeAssistant do not show-up in HomeBridge as light switch

### ISSUE THREAD

**Issue by B1tMaster** [home-assistant/homebridge-homeassistant#154]
Hi,

Not sure if this is a problem of this plugin or Homebridge itself. But it seems that at the moment, "media-player" no longer showing up in HomeKit app on iPhone. 

I have both homebridge and homeassistant installed and configured. They see each other and logs below show events coming from home assistant to homebridge. (from homebridge side)

But media player supposed to auto show up in HomeKit, and that does not happen.

Any idea on how to debug this further? 

```
May 07 10:02:01 HomeControl homebridge[710]: 79+00:00", "entity_id": "media_player.study"}, "entity_id": "media_player.study"}, "event_type": "state_changed"}
May 07 10:02:01 HomeControl homebridge[710]: [5/7/2017, 10:02:01 AM] [HomeAssistant] Received event: {"origin": "LOCAL", "time_fired": "2017-05-07T02:02:01.217978+00:00", "data": {"new_state": {"attributes": {"media_content_id": "WBBRAMAAC_SC?TGT=TuneIn&DIST=TuneIn", "source": "Bloomberg Radio", "media_title": "Bloomberg Radio", "is_coordinator": true, "supported_features": 27150, "media_artist": "Bloomberg Law", "media_album_name": "", "source_list": ["BBC Radio 4", "BBC World Service News", "Bloomberg Radio", "Chill", "FOX News Talk", "Long Ambients1: Calm. Sleep.", "NPR Program Stream", "Ocean by Ocean", "Radio Kansas", "Sleep Sounds", "WAMU", "WNPR", "\u041d\u0430\u0448\u0435 \u0420\u0430\u0434\u0438\u043e", "Line-in"], "media_content_type": "music", "entity_picture": "/api/media_player_proxy/media_player.study?token=fe0a98ddcbb9885e6a133ecc3da6ae005d2ce43c1a73c74c2cca1208867d8e82&cache=41def", "media_duration": 0, "friendly_name": "Study", "is_volume_muted": false, "volume_level": 0.5}, "last_changed": "2017-05-06T16:37:56.907510+00:00", "state": "paused", "last_updated": "2017-05-07T02:02:01.217838+00:00", "entity_id": "media_player.study"}, "old_state": {"attributes": {"media_content_id": "x-sonosapi-stream:s165740?sid=254&flags=32&sn=0", "source": "Bloomberg Radio", "source_list": ["BBC Radio 4", "BBC World Service News", "Bloomberg Radio", "Chill", "FOX News Talk", "Long Ambients1: Calm. Sleep.", "NPR Program Stream", "Ocean by Ocean", "Radio Kansas", "Sleep Sounds", "WAMU", "WNPR", "\u041d\u0430\u0448\u0435 \u0420\u0430\u0434\u0438\u043e", "Line-in"], "media_title": "Bloomberg Radio", "media_artist": "Bloomberg Law", "supported_features": 27150, "is_coordinator": true, "media_content_type": "music", "entity_picture": "/api/media_player_proxy/media_player.study?token=fe0a98ddcbb9885e6a133ecc3da6ae005d2ce43c1a73c74c2cca1208867d8e82&cache=41def", "media_duration": 0, "friendly_name": "Study", "is_volume_muted": false, "volume_level": 0.5}, "last_changed": "2017-05-06T16:37:56.907510+00:00", "state": "paused", "last_updated": "2017-05-07T02:01:59.4777
```

**Comment by B1tMaster** [comment]
I managed to find a work around.. Maybe it is after all problem with this plugin.. to make things work I basically just moved "media_plugin" to be first in the list. And things started working..

So from this:  

```
"platforms": [
     {
    "platform": "HomeAssistant",
    "name": "HomeAssistant",
    "host": "http://localhost:8123",
    "password": "YOURPASSWORD.",
    "supported_types": [ "binary_sensor", "climate", "cover", "device_tracker", "fan", "group", "input_boolean", "light", "lock","scene", "media_player", "sensor", "switch"],
    "logging": true
    }
    ]

```

It became this: (note how "media_player" moved to the front of the list)

```
"platforms": [
     {
    "platform": "HomeAssistant",
    "name": "HomeAssistant",
    "host": "http://localhost:8123",
    "password": "YOURPASSWORD.",
    "supported_types": ["media_player", "binary_sensor", "climate", "cover", "device_tracker", "fan", "group", "input_boolean", "light", "lock","scene", "sensor", "switch"],
    "logging": true
    }
    ]
```

**Comment by robbiet480** [comment]
I believe this is fixed via #183, closing as such, please re-open if I was wrong!
